### DISCUSSION TITLE
v2.10.31

### DISCUSSION THREAD

**Discussion by fiftin** [semaphoreui/semaphore#2441] [Announcements]
## Bugfixes
* Fix ldapmapping panic https://github.com/semaphoreui/semaphore/pull/2440 @DarioCalovic
* The task log is not displayed https://github.com/semaphoreui/semaphore/issues/1968 https://github.com/semaphoreui/semaphore/pull/2434 @kovalewvladimir 
* Problem: Running Tasks not working anymore https://github.com/semaphoreui/semaphore/issues/2428 https://github.com/semaphoreui/semaphore/pull/2429 @0x4D616E75 

## Features
* Allow override of Task and Template git_branch https://github.com/semaphoreui/semaphore/pull/2278 @Omicron7 
* Add Ukrainian language support https://github.com/semaphoreui/semaphore/pull/2438 @click0 

<hr /><em>This discussion was created from the release <a href='https://github.com/semaphoreui/semaphore/releases/tag/v2.10.31'>v2.10.31</a>.</em>

**Comment by click0** [comment]
What about the patch [Corrected translation into Russian. #2432](https://github.com/semaphoreui/semaphore/pull/2432)?
He's not good enough? :)

**Reply by fiftin** [reply]
I'm not sure that Semaphore should be translated to Russian. 

Semaphore UI - is a name of the app, it can't be translated as "Интерфейс пользователя Семафора".

**Reply by click0** [reply]
This is from the main page of the project:
```
Semaphore UI

Modern UI for Ansible, Terraform, OpenTofu, PowerShell and other DevOps tools.
```
and
```
What is Semaphore UI?

Semaphore UI is a modern web interface for managing popular DevOps tools.
```

What is `UI` then? Maybe User Interface?

Literal Russian translation - 'Интерфейс пользователя "Семафор"'.
Well, you can also use this option - 'Автоматизированный программный комплекс "Семафор"'

**Reply by click0** [reply]
Apart from the complaint about the line
https://github.com/semaphoreui/semaphore/pull/2432/commits/9caf016a39e7fe006565981d5a181926fc5e23c5#L81
are there any other complaints about the translation?

And in general, this is what сode review is for, where you can express your wishes for each line of the commit.
